nmpolicy/internal/capture/capture.go

```diff
@@ -88,6 +88,12 @@ func (c Capture) Resolve(
 	return capturesState, nil
 }
 
+func (c Capture) ResolveCapturePath(
+	capturesCache map[string]types.CaptureState,
+	capturePath string) (interface{}, error) {
+	return nil, nil
+}
```

Comment:
```suggestion
func (c Capture) ResolveCaptureEntryPath(captureEntryPath string) (interface{}, error) {
	return nil, nil
}
```

The capturedState is already resolved inside the  `Resolver` struct living inside the `Capture` so we just need to pass the `captureEntryPath` string.

Note that for this to work Resolve has to be called before this one.